Attempt at fixing issue #115

micokoch opened the pull request:
Hello,
This PR was trying to address issue #115 . As I mentioned in the issue, I used the HubEnsembles manuscript verbiage to explain the model output and maintain consistency. @elray1, @nickreich, @lshandross - could you look at this PR and see if it can be approved / needs changes / needs to be discarded.

As a side note, I'm having issues because my fork keeps falling behind, and some of the new changes are incompatible, and it has been a bit of a nightmare to sync my fork and branches.
Thank you!

lshandross commented on docs/source/user-guide/model-output.md:
```diff
@@ -49,10 +58,10 @@ We emphasize that the `mean`, `median`, `quantile`, `cdf`, and `pmf` representat
 4. the marginal predictive distribution for each combination of forecast date, location, and horizon
 
-Hubs should specify the collection of task id variables for which samples are expected to capture dependence; e.g., the first option listed above might specify that samples should be drawn from distributions that are “joint across” locations and horizons.
+Hubs should specify the collection of task id variables for which samples are expected to capture dependence; e.g., the first option listed above might specify that samples should be drawn from distributions that are “joint across” locations and horizons.  
```
Is there a reason why there's extra whitespace at the end of the line here and on line 64?

micokoch replied:
When I remember, I try to leave to white spaces after every line, because it ensures there is a line break before the next line. It's usually not necessary, but also serves as a marker that there should be a line break.

lshandross replied:
Linting rules generally seem to prefer no whitespace at the end of lines, though I'm not sure what the rules are for non-code files. Since there's already a new line below, the whitespace probably isn't necessary, but I don't have a strong opinion on this. I'll approve the changes for now since I don't have any other comments.

lshandross commented on docs/source/user-guide/sample-output-type.md:
```diff
@@ -1,309 +1,604 @@
+In the hubverse, a “modeling task” is the element that is being predicted and that can be represented by a univariate (e.g., scalar, or single) value. We could also tie this to a tabular representation of data more concretely as a combination of values from a set of task id columns that uniquely define a single prediction. We note that this concept is similar to that of a [“forecast unit” in the  ...
+
+Take the following model_output data for the mean output_type as an example:
+| Origin_date | horizon | location | output_type| Output_type_id | value |
```
Is there a reason that `Origin_Date` and `Output_type_id` are capitalized?

micokoch replied:
Hi @lshandross - thanks for reviewing this so carefully. As far as I can tell, all those edits on `sample-output-type` were mistakes made when I tried to update my fork and so forth. What I did was just copy the newest official version of `sample-output-type` and replace it with my old copy. So all these issues should be solved now.

lshandross commented on docs/source/user-guide/sample-output-type.md:
```diff
@@ -1,309 +1,604 @@
+## Individual modeling tasks
+In many settings, forecasts will be made for individual modeling tasks, with no notion of modeling tasks being related to each other or collected into sets (for more on this, see [Compound modeling tasks](https://github.com/Infectious-Disease-Modeling-Hubs/hubDocs/edit/mzorn-patch-3/docs/source/user-guide/sample%20output_type.md#compound-modeling-tasks)). In the situations where forecasts are ass ...
+
+|compound_idx| Origin_date | horizon | location | output_type| Output_type_id | value |
```
Again, Origin_Date and Output_type_id are capitalized here

micokoch replied:
Hi @lshandross - thanks for reviewing this so carefully. As far as I can tell, all those edits on `sample-output-type` were mistakes made when I tried to update my fork and so forth. What I did was just copy the newest official version of `sample-output-type` and replace it with my old copy. So all these issues should be solved.

lshandross commented on docs/source/user-guide/sample-output-type.md:
```diff
@@ -1,309 +1,604 @@
+		"output_type_id_params":{
+			“is_required”: true,
+			“type”: “integer”,
+"min_samples_per_task": 100,
```
Fix the indentation here

micokoch replied:
Hi @lshandross - thanks for reviewing this so carefully. As far as I can tell, all those edits on `sample-output-type` were mistakes made when I tried to update my fork and so forth. What I did was just copy the newest official version of `sample-output-type` and replace it with my old copy. So all these issues should be solved.

lshandross commented on docs/source/user-guide/sample-output-type.md:
```diff
@@ -1,309 +1,604 @@
+
+Base data: mean output_type. In the table below, an entry of “-” stands in for specific values to be provided by the submitter.
+
+| Origin_date | horizon | variant |location | output_type| Output_type_id | value |
```
Again, Origin_Date and Output_type_id are capitalized here

micokoch replied:
Hi @lshandross - thanks for reviewing this so carefully. As far as I can tell, all those edits on `sample-output-type` were mistakes made when I tried to update my fork and so forth. What I did was just copy the newest official version of `sample-output-type` and replace it with my old copy. So all these issues should be solved.

lshandross commented on docs/source/user-guide/sample-output-type.md:
```diff
@@ -1,309 +1,604 @@
+			“is_required”: true,
+			“type”: “character”,
+			“max_length”: 6,
+"min_samples_per_task": 90,
```
Fix indentation here

micokoch replied:
Hi @lshandross - thanks for reviewing this so carefully. As far as I can tell, all those edits on `sample-output-type` were mistakes made when I tried to update my fork and so forth. What I did was just copy the newest official version of `sample-output-type` and replace it with my old copy. So all these issues should be solved.

lshandross commented on docs/source/user-guide/sample-output-type.md:
```diff
@@ -1,309 +1,604 @@
+
+<div class="heatMap1">
+
+|compound_idx| Origin_date |location | horizon | variant | output_type| Output_type_id | value |
```
Again, Origin_Date and Output_type_id are capitalized here

micokoch replied:
Hi @lshandross - thanks for reviewing this so carefully. As far as I can tell, all those edits on `sample-output-type` were mistakes made when I tried to update my fork and so forth. What I did was just copy the newest official version of `sample-output-type` and replace it with my old copy. So all these issues should be solved.

lshandross commented on docs/source/user-guide/sample-output-type.md:
````diff
@@ -1,309 +1,604 @@
-```
-
-```{attention}
-Rows are shaded to indicate different samples for the same compound forecast task. For example in the table below, the first two rows correspond to the same modeling task but they are highlighted to show that the first row is from sample "s0" while the second row is from sample "s1".
````
Is there a reason this part was removed? It seems like it may be helpful to the reader

micokoch replied:
Hi @lshandross - thanks for reviewing this so carefully. As far as I can tell, all those edits on `sample-output-type` were mistakes made when I tried to update my fork and so forth. What I did was just copy the newest official version of `sample-output-type` and replace it with my old copy. So all these issues should be solved.

lshandross commented on docs/source/user-guide/sample-output-type.md:
```diff
@@ -1,309 +1,604 @@
+			“is_required”: true,
+			“type”: “character”,
+			“max_length”: 6,
+"min_samples_per_task": 90,
```
Fix indentation here

micokoch replied:
Hi @lshandross - thanks for reviewing this so carefully. As far as I can tell, all those edits on `sample-output-type` were mistakes made when I tried to update my fork and so forth. What I did was just copy the newest official version of `sample-output-type` and replace it with my old copy. So all these issues should be solved.

lshandross commented on docs/source/user-guide/sample-output-type.md:
```diff
@@ -1,309 +1,604 @@
+			“is_required”: true,
+			“type”: “character”,
+			“max_length”: 6,
+"min_samples_per_task": 90,
```
Fix indentation here

micokoch replied:
Hi @lshandross - thanks for reviewing this so carefully. As far as I can tell, all those edits on `sample-output-type` were mistakes made when I tried to update my fork and so forth. What I did was just copy the newest official version of `sample-output-type` and replace it with my old copy. So all these issues should be solved.

lshandross commented on docs/source/user-guide/sample-output-type.md:
```diff
@@ -1,309 +1,604 @@
+
+<div class="heatMap3">
+
+|compound_idx| Origin_date |location | horizon | variant | output_type| Output_type_id | value |
```
Again, Origin_Date and Output_type_id are capitalized here

micokoch replied:
Hi @lshandross - thanks for reviewing this so carefully. As far as I can tell, all those edits on `sample-output-type` were mistakes made when I tried to update my fork and so forth. What I did was just copy the newest official version of `sample-output-type` and replace it with my old copy. So all these issues should be solved.

lshandross commented on docs/source/user-guide/sample-output-type.md:
```diff
@@ -1,309 +1,604 @@
+			“is_required”: true,
+			“type”: “character”,
+			“max_length”: 6,
+"min_samples_per_task": 90,
```
Fix indentation here

micokoch replied:
Hi @lshandross - thanks for reviewing this so carefully. As far as I can tell, all those edits on `sample-output-type` were mistakes made when I tried to update my fork and so forth. What I did was just copy the newest official version of `sample-output-type` and replace it with my old copy. So all these issues should be solved.

lshandross commented on docs/source/user-guide/sample-output-type.md:
```diff
@@ -1,309 +1,604 @@
+
+<div class="heatMap4">
+
+|compound_idx| Origin_date |location | horizon | variant | output_type| Output_type_id | value |
```
Again, Origin_Date and Output_type_id are capitalized here

micokoch replied:
Hi @lshandross - thanks for reviewing this so carefully. As far as I can tell, all those edits on `sample-output-type` were mistakes made when I tried to update my fork and so forth. What I did was just copy the newest official version of `sample-output-type` and replace it with my old copy. So all these issues should be solved.

lshandross commented on docs/source/user-guide/sample-output-type.md:
```diff
@@ -1,309 +1,604 @@
+
+At a later time, the hubverse may revisit a way to more generally define compound modeling tasks, as they can be used for different things. For example, compound modeling tasks defined for a compositional data target could
+
+<li>validation that all of the proportions in a set of “mean” output_types sum to 1.</li>
```
```suggestion
<li>validate that all of the proportions in a set of “mean” output_types sum to 1.</li>
```

micokoch replied:
Hi @lshandross - thanks for reviewing this so carefully. As far as I can tell, all those edits on `sample-output-type` were mistakes made when I tried to update my fork and so forth. What I did was just copy the newest official version of `sample-output-type` and replace it with my old copy. So all these issues should be solved.

lshandross commented on docs/source/user-guide/sample-output-type.md:
```diff
@@ -1,309 +1,604 @@
+
+At a later time, the hubverse may revisit a way to more generally define compound modeling tasks, as they can be used for different things. For example, compound modeling tasks defined for a compositional data target could
+
+ - validation that all of the proportions in a set of “mean” output_types sum to 1.
```
```suggestion
 - validate that all of the proportions in a set of “mean” output_types sum to 1.
```

micokoch replied:
Hi @lshandross - thanks for reviewing this so carefully. As far as I can tell, all those edits on `sample-output-type` were mistakes made when I tried to update my fork and so forth. What I did was just copy the newest official version of `sample-output-type` and replace it with my old copy. So all these issues should be solved.

lshandross commented on docs/source/user-guide/sample-output-type.md:
```diff
@@ -1,309 +1,604 @@
+
+<br>
+
+Documentation about tests for other output types can be found in  [Validation Pull Requests on GitHub](https://infectious-disease-modeling-hubs.github.io/hubValidations/articles/validate-pr.html#validate_pr-check-details).  
```
```suggestion
Documentation about tests for other output types can be found in [Validation Pull Requests on GitHub](https://infectious-disease-modeling-hubs.github.io/hubValidations/articles/validate-pr.html#validate_pr-check-details).  
```

micokoch replied:
Hi @lshandross - thanks for reviewing this so carefully. As far as I can tell, all those edits on `sample-output-type` were mistakes made when I tried to update my fork and so forth. What I did was just copy the newest official version of `sample-output-type` and replace it with my old copy. So all these issues should be solved.

micokoch commented:
Hi @lshandross - thanks for the review. Sorry I made you review a problematic version of `sample-output-type`. I messed up when I was updating my fork and I didn't even realize I had kept that old version of the file. I just deleted my version and updated it with the current, accepted one - so that should all be fixed. Thanks!

micokoch commented:
Hello. I am removing @nickreich as a reviewer, since he reviewed this previously and I made the change he asked for (removing the journal title).
